src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -2221,5 +2221,32 @@ class AppsResourceTest extends AkkaUnitTest with GroupCreation with JerseyTest {
       }
     }
 
+    "Create a new app inside a top-group with enforceRole applies the proper group-role default" in new FixtureWithRealGroupManager(initialRoot = createRootGroup(groups = Set(createGroup("/dev".toPath, enforceRole = true)))) {
+      service.deploy(any, any) returns Future.successful(Done)
+
+      Given("group /dev with enforceRole: true")
+
+      When("I post a sleeper app definition without a leading slash to dev/sleeper")
+      val response = asyncRequest { r =>
```

Comment:
This test failed before applying this fix

Reply:
Cool. What line is the actual fix?